server/common/common/urls.py

```diff
@@ -21,7 +21,7 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('api/v1/',include('api.urls'))
+    path('api/v1/',include('api.urls')),
```

Comment:
Do we need this comma in here?
I think that this element is the last elements of a list, so we do not need the comma.

Reply:
I edit it